onlinebanking/src/main/java/com/adyen/checkout/onlinebanking/OnlineBankingPLConfiguration.kt

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2022 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by atef on 10/8/2022.
+ */
+
+package com.adyen.checkout.onlinebanking
+
+import android.content.Context
+import android.os.Parcel
+import android.os.Parcelable
+import com.adyen.checkout.core.api.Environment
+import com.adyen.checkout.issuerlist.IssuerListConfiguration
+import java.util.Locale
+
+class OnlineBankingPLConfiguration : IssuerListConfiguration {
+
+    internal constructor(
```

Comment:
Shouldn't these constructors be `private`?

Reply:
Yes, you're right I just copied and pasted it from `IdealConfiguration`